### ISSUE TITLE
Mark account only for testnet

### ISSUE THREAD

**Issue by xlc** [polkadot-js/extension#461]
We can mark account only use for certain networks.
It will useful to mark account only use for development. i.e. network with prefix 42

**Comment by Tbaut** [comment]
I feel this might end up looking convoluted for users if we mix "testnet" with network names.

**Comment by xlc** [comment]
How about add `Generic Substrate`?

<img width="221" alt="Screen Shot 2020-09-21 at 9 36 40 PM" src="https://user-images.githubusercontent.com/1201310/93752276-8a896780-fc52-11ea-8228-f5218d67e666.png">

**Comment by Tbaut** [comment]
I totally understand your point, I do have such accounts allowed on any chain as well, I name them "TEST *" to make sure I don't mess up.
"Substrate generic (testnet)" could indeed by a good middle ground IMO. I'd add this depending on @jacogr opinion.

**Comment by jacogr** [comment]
The above dropdown add the `genesisHash` for the selected account, i.e. so it is only useful on that network. There is no "generic substrate genesisHash" and actually the extension is completely oblivious to what may or may not be a test network.

**Comment by xlc** [comment]
In that case is there a way to mark an account not to be used by a known network?

- [ ] Polkadot
- [ ] Kusama
- [ ] Crab
- [ ] Others

**Comment by jacogr** [comment]
No, however both genesisHash-allowed/blocked (as a list, not just a single) is interesting for the keyring component.

**Comment by xlc** [comment]
Able to block a list of known networks will work as well.

**Comment by Tbaut** [comment]
One thing that I've missed recently, is the ability to set the network for an account for Westedn, which I have the metadata, but that doesn't appear in the list.
I think this might solve at least part of your issue.

**Comment by jacogr** [comment]
This issue has been open for 21 days with no activity and is not labelled as an enhancement. It will be closed in 7 days.

**Comment by ap211unitech** [comment]
This issue appears to be outdated. However, feel free to open a new issue with updated details if you're still experiencing the problem.
